### ISSUE TITLE
您好，我目前按照您说的方式，成功通过您给的数据集中的graph文件中的json文件画出了极点图， 并且将json文件中的012分别赋予了不同的颜色。接下来我试着将rgb图片输入训练好的模型，成功输出了3*16*27的结果，并且我将他们保存到json文件中。 但是查看的时候，我发现得到的结果全部都是小数。请问我该怎么处理这些小数来输出极点图呢？我试着将3*16*27的结果沿着第一个维度求和，但是结果依然是小数。 请问您是通过什么方式将他们归一化的呢？

### ISSUE THREAD

**Issue by 1146065545** [lab-sun/PolarPoint-BEV#4]

**Comment by 1146065545** [comment]
import os
import torch
from PIL import Image
import json
import numpy as np
from model2 import XPlan
from torchvision import transforms as T



device = torch.device('cpu')

model = XPlan().to(device)

ckpt = torch.load('XPlan-N.ckpt', map_location=torch.device('cpu'))
model.load_state_dict(ckpt["state_dict"], strict=False)
model.eval()


image_folder = 'rgb'
output_folder = 'graph_pre_XPLAN'
_im_transform = T.Compose([T.ToTensor(), T.Normalize(mean=[0.485,0.456,0.406], std=[0.229,0.224,0.225])])

if not os.path.exists(output_folder):
    os.makedirs(output_folder)


for image_name in os.listdir(image_folder):
    if image_name.endswith('.png'):

        image_path = os.path.join(image_folder, image_name)
        img = Image.open(image_path)

        img_tensor = _im_transform(np.array(img)).unsqueeze(0)


        with torch.no_grad():
            pred = model(img_tensor).squeeze(0)  

        output_data = pred.tolist()


        json_path = os.path.join(output_folder, f'{os.path.splitext(image_name)[0]}.json')
        with open(json_path, 'w') as json_file:
            json.dump(output_data, json_file)
